src/main/java/seedu/address/logic/parser/CliSyntaxPrescription.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.logic.parser;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple commands
+ */
+public class CliSyntaxPrescription {
+
+    /* Prefix definitions */
+    public static final Prefix PREFIX_NAME = new Prefix("mn/");
+    // public static final Prefix PREFIX_TAG = new Prefix("t/");
```

Comment:
any reason why we are preserving tag? For additional info in the future?

Reply:
yup, in case we want to make notes multi valued like tag